iset.mm

```diff
@@ -147339,6 +147339,448 @@ since the target of the homomorphism (operator ` O ` in our model) need
+
+    $( Commutative/associative subtraction law for Abelian groups.
+       (Contributed by NM, 31-Mar-2014.) $)
+    ablsub4 $p |- ( ( G e. Abel /\ ( X e. B /\ Y e. B ) /\ ( Z e. B /\ W e. B )
+        ) -> ( ( X .+ Y ) .- ( Z .+ W ) ) = ( ( X .- Z ) .+ ( Y .- W ) ) ) $=
+      ( wcel wa co cfv wceq 3ad2ant1 syl3anc grpsubval syl2anc cabl w3a cminusg
+      cgrp ablgrp simp2l simp2r grpcl simp3l simp3r eqid ccmn ablcmn simp2 cmn4
+      grpinvcl syl112anc simp1 ablinvadd oveq2d oveq12d 3eqtr4d eqtrd ) CUALZFA
+      LZGALZMZHALZEALZMZUBZFGBNZHEBNZDNZVLVMCUCOZOZBNZFHDNZGEDNZBNZVKVLALZVMALZ
+      VNVQPVKCUDLZVEVFWAVDVGWCVJCUEQZVDVEVFVJUFZVDVEVFVJUGZABCFGIJUHRVKWCVHVIWB
+      WDVDVGVHVIUIZVDVGVHVIUJZABCHEIJUHRABCVODVLVMIJVOUKZKSTVKVLHVOOZEVOOZBNZBN
+      ZFWJBNZGWKBNZBNZVQVTVKCULLZVGWJALZWKALZWMWPPVDVGWQVJCUMQVDVGVJUNVKWCVHWRW
+      DWGACVOHIWIUPTVKWCVIWSWDWHACVOEIWIUPTABCWKFGWJIJUOUQVKVPWLVLBVKVDVHVIVPWL
+      PVDVGVJURWGWHABCVOHEIJWIUSRUTVKVRWNVSWOBVKVEVHVRWNPWEWGABCVODFHIJWIKSTVKV
+      FVIVSWOPWFWHABCVODGEIJWIKSTVAVBVC $.
+
+    $( Abelian group addition/subtraction law.  (Contributed by NM,
+       31-Mar-2014.) $)
+    abladdsub4 $p |- ( ( G e. Abel
+       /\ ( X e. B /\ Y e. B ) /\ ( Z e. B /\ W e. B ) )
+   -> ( ( X .+ Y ) = ( Z .+ W ) <-> ( X .- Z ) = ( W .- Y ) ) ) $=
+      ( wcel wa co wceq grpcl syl3anc ablsub4 syl122anc syl2anc cabl w3a ablgrp
+      wb 3ad2ant1 simp2l simp2r simp3l simp3r grpsubrcan syl13anc c0g cfv simp1
+      cgrp grpsubid oveq2d grpsubcl grprid 3eqtrd oveq1d grplid eqeq12d bitr3d
+      eqid ) CUALZFALZGALZMZHALZEALZMZUBZFGBNZHGBNZDNZHEBNZVODNZOZVNVQOZFHDNZEG
+      DNZOVMCUOLZVNALZVQALZVOALZVSVTUDVFVIWCVLCUCUEZVMWCVGVHWDWGVFVGVHVLUFZVFVG
+      VHVLUGZABCFGIJPQVMWCVJVKWEWGVFVIVJVKUHZVFVIVJVKUIZABCHEIJPQVMWCVJVHWFWGWJ
+      WIABCHGIJPQACDVNVQVOIKUJUKVMVPWAVRWBVMVPWAGGDNZBNZWACULUMZBNZWAVMVFVGVHVJ
+      VHVPWMOVFVIVLUNZWHWIWJWIABCDGFGHIJKRSVMWLWNWABVMWCVHWLWNOWGWIACDGWNIWNVEZ
+      KUPTUQVMWCWAALZWOWAOWGVMWCVGVJWRWGWHWJACDFHIKURQABCWAWNIJWQUSTUTVMVRHHDNZ
+      WBBNZWNWBBNZWBVMVFVJVKVJVHVRWTOWPWJWKWJWIABCDGHEHIJKRSVMWSWNWBBVMWCVJWSWN
+      OWGWJACDHWNIWQKUPTVAVMWCWBALZXAWBOWGVMWCVKVHXBWGWKWIACDEGIKURQABCWBWNIJWQ
+      VBTUTVCVD $.
+
+    $( Associative-type law for group subtraction and addition.  (Contributed
+       by NM, 19-Apr-2014.) $)
+    abladdsub $p |- ( ( G e. Abel /\ ( X e. B /\ Y e. B /\ Z e. B ) ) ->
+         ( ( X .+ Y ) .- Z ) = ( ( X .- Z ) .+ Y ) ) $=
+      ( cabl wcel w3a wa co wceq ablcom 3adant3r3 oveq1d syl3anc syl13anc simpl
+      cgrp ablgrp adantr simpr2 simpr1 simpr3 grpaddsubass grpsubcl 3eqtrd ) CK
+      LZEALZFALZGALZMZNZEFBOZGDOFEBOZGDOZFEGDOZBOZVAFBOZUQURUSGDULUMUNURUSPUOAB
+      CEFHIQRSUQCUCLZUNUMUOUTVBPULVDUPCUDUEZULUMUNUOUFZULUMUNUOUGZULUMUNUOUHZAB
+      CDFEGHIJUIUAUQULUNVAALZVBVCPULUPUBVFUQVDUMUOVIVEVGVHACDEGHJUJTABCFVAHIQTU
+      K $.
+
+    $( Cancellation law for subtraction.  (Contributed by NM, 2-Oct-2014.) $)
```

Comment:
Cancellation law for subtraction in an Abelian group.

Reply:
Changed in set.mm and iset.mm.

Although we don't always spell out every detail in comments, there are enough cancellation laws that it seems worthwhile to say this is for Abelian groups.